### ISSUE TITLE
RouterView is not correctly populating when changing route

### ISSUE THREAD

**Issue by davidangelesu** [Kitware/trame-router#3]
Hi there,

I am having some issues with the RouterViewer as it is not populating the correct view (apart from the view linked to the root "/" path)

![image](https://github.com/Kitware/trame-router/assets/8888658/e476307f-49bc-4261-81ee-5a3a543bb8cf)

I am unable to pinpoint the reason as to what is causing this issue.

I am utilizing _vue3_ with the following version of the packages:
```
trame 3.5.4
trame-client 2.16.4
trame-router 2.2.0
trame-server 2.17.2
trame-vuetify 2.4.3
```

I am using the same same source code as the one shown in the package readme.md

**Comment by jourdain** [comment]
Can you provide a small example illustrating the issue? Thx

**Comment by davidangelesu** [comment]
Hi, sure:
I am providing here a link to the main file of my repo with a simple example. (I also added the requirement.txt files to  be able to replicate it)

https://github.com/davidangelesu/FWI-Visualization/blob/basic-router-example/main.py

**Comment by jourdain** [comment]
fixed in `trame-client==2.16.5`
